MAINTAINING.md

```diff
@@ -0,0 +1,23 @@
+# Maintaining UMA Repositories
```

Comment:
do you want to add a section that mentions how we cycle between weeks of on-call for maintaining? i.e "each eng is on-call for one week in a round robbin fashion" or should this be left off gh?

Reply:
I don't know if it should be managed off-github, but I think this doc shouldn't prescribe the rotation. IMO, that should be managed in whatever tool/doc we use to actually do the rotation. I think this doc should just be able to point to somewhere you can go to figure out who the current on-call maintainer is.